src/pages/kb/data-sources/querying-mongodb.md

```diff
@@ -100,8 +102,13 @@ We support  [MongoDB Extended JSON](https://docs.mongodb.com/manual/reference/mo
     }
     
 
-It accepts a human-readable string like the above (“3 years ago”, “yesterday”,
-etc) or timestamps.
+It accepts a human-readable string like the above (“3 years ago”, “yesterday”, etc) or timestamps.
+
+{% callout info %}
+
+The `$humantime` function is needed when using [Query Parameters]({% link _kb/user-guide/querying/query-parameters.md %}) of type Date with MongoDB. If your Date query parameters do not behave as expected, wrap them in `$humantime` calls (`"{{param}}"` would be converted to `{"$humanTime": "{{ param }}"}`). This is necessary because Date type Query Parameters do not meet MongoDB's date format requirements.
```

Comment:
```suggestion
The `$humanTime` function is also needed when using [Query Parameters]({% link _kb/user-guide/querying/query-parameters.md %}) of type Date or Date/Time with MongoDB, due to the difference between the format Redash uses and the one MongoDB expects. 

When using a Date (or Date Range) parameter, wrap it with a `$humanTime` object: `{{param}}` becomes `{"$humanTime": "{{param}} 00:00"}` (the ` 00:00` suffix is needed only with Date parameters, for Date Time parameters you should skip it).
```

Note that it's `$humanTime` with a capital _t_ and the comment about adding '00:00'.